simulation/constants.py

```diff
@@ -0,0 +1,56 @@
+import enum
+from typing import Dict
+
+
+class PropertyType(enum.Enum):
+    HOUSE = 0
+    FLAT = 1
+    BUNGALOW = 2
+
+
+class BuiltForm(enum.Enum):
+    MID_TERRACE = 0
+    SEMI_DETACHED = 1
+    DETACHED = 2
+    END_TERRACE = 3
+
+
+class OccupantType(enum.Enum):
+    OWNER_OCCUPIER = 0
+    RENTER_PRIVATE = 1
+    RENTER_SOCIAL = 1
```

Comment:
```suggestion
    RENTER_SOCIAL = 2
```
Did you want these all to be different or actually the 0 or 1 so that when we convert to `OccupantType.value` we essentially get `is_renter` ?
To be honest, I quite like the latter approach - just wanted to check it was intentional here?

Reply:
I think that'll be easy to miss. If we don't need the private/social distinction remove it otherwise make them different values.

Reply:
This was a typo - I intended them to represent different values. While the distinction between renters does not alter agent behaviour, but is potentially a field we would want to segment simulation outputs by, so I'd prefer not to lose this detail.